airgun/views/cloud_insights.py

```diff
@@ -47,9 +49,9 @@ class CloudInsightsView(BaseLoggedInView, SearchableViewMixin):
     """Main RH Cloud Insights view."""
 
     title = Text('//h1[text()="Red Hat Insights"]')
-    insights_sync_switcher = InventoryBootstrapSwitch(class_name='insights_sync_switcher')
-    start_hits_sync = Button('Start recommendations sync')
+    insights_sync_switcher = Switch('OUIA-Generated-Switch-1')
     remediate = Button('Remediate')
+    insights_dropdown = Dropdown(locator='.//div[contains(@class, "title-dropdown")]')
```

Comment:
No OUIA attribute for the div element here?  No need to import the OUIA version of the widget if we're not using the relevant locator.

Reply:
@jameerpathan111 sorry, I read your import wrong, you can ignore this comment. Thanks for reaching out.